src/test/suite/connectionController.test.ts

```diff
@@ -984,17 +1001,13 @@ suite('Connection Controller Test Suite', function () {
       storageLocation: StorageScope.NONE
     };
 
-    const originalConnect = DataService.connect;
```

Comment:
DataService is a function and `DataService.connect` is always undefined. I have rewritten this part a bit to get access to the connect function.